src/ark/utils/deepcell_service_utils.py

```diff
@@ -18,10 +17,91 @@
 from urllib3 import Retry
 
 
+def zip_input_files(deepcell_input_dir, fov_group, batch_num):
+    """Helper function which handles zipping the batch fov images into a single zip file.
+
+    Args:
+        deepcell_input_dir (str): path to where deepcell input image files are stored
+        fov_group (list): list of fovs to process in this batch
+        batch_num (int): the batch number
+    Returns:
+        str: path to deepcell input zip file
+    """
+
+    # write all files to the zip file
+    zip_path = os.path.join(deepcell_input_dir, f'fovs_batch_{batch_num + 1}.zip')
+
+    # create zip files, skip any existing
+    if not os.path.exists(zip_path):
+        print('Zipping preprocessed tiff files.')
+        with ZipFile(zip_path, 'w', compression=ZIP_DEFLATED) as zipObj:
+            for fov in fov_group:
+                # file has .tiff extension
+                basename = fov + '.tiff'
+                filename = os.path.join(deepcell_input_dir, basename)
+                zipObj.write(filename, basename)
```

Comment:
I'd leave the option to overwrite `.zip` files here. Or at the very least, warn the user that since the `.zip` file exists, it won't be written to.